package.json

```diff
@@ -40,10 +39,12 @@
     "lib/transport-utils.js"
   ],
   "devDependencies": {
-    "seneca": "plugin",
+    "async": "1.4.2",
+    "bench": "^0.3.5",
     "docco": "0.7.0",
     "jshint": "2.8.0",
     "mocha": "2.1.0",
+    "seneca": "plugin",
     "seneca-transport-test": "0.1.3"
```

Comment:
good idea.  We can wait to publish any update to seneca-transport until we have confirmed against the latest seneca-transport-test